src/CitizenProject/CitizenProjectMessageNotifier.php

```diff
@@ -65,6 +72,19 @@ private function scheduleCreationNotification(CitizenProject $citizenProject): v
         ]));
     }
 
+    private function sendAskCommitteeSupport(CitizenProject $citizenProject, Adherent $creator): void
+    {
+        foreach ($citizenProject->getPendingCommitteeSupports() as $committeeSupport) {
+            $this->mailer->sendMessage(
+                CitizenProjectRequestCommitteeSupportMessage::create(
+                    $citizenProject,
+                    $creator,
```

Comment:
Passing the creator all along is not needed, just call `$citizenProject->getCreator()` in that `create` method.